docker/install/ubuntu_install_papi.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+set -o pipefail
+
+apt-get update --fix-missing
+
+# deps
+apt-get install -y linux-tools-common linux-tools-generic
+
+cd /
+git clone https://bitbucket.org/icl/papi.git
+cd papi
+# We need a fix for cuda instantiation on master
```

Comment:
i think for the docker container deps, we prefer to let them update as we rebuild, so we don't get stale. so do we need this, considering it's checked-in to master now?

Reply:
switched to head.